packages/manager/modules/nasha/src/dashboard/partition/partition.template.html

```diff
@@ -3,25 +3,14 @@
         <div class="d-flex pt-4">
             <h1
                 class="word-break mb-0 oui-header-2"
-                data-ng-bind="$ctrl.name"
+                data-ng-bind="$ctrl.partition.partitionName"
             ></h1>
-            <button
-                type="button"
-                class="btn btn-icon my-auto ml-2"
-                data-translate-attr="{ title: 'nasha_dashboard_partition_edit' }"
-                data-ng-click="$ctrl.openEditNameModal()"
-            >
-                <span
-                    class="oui-icon oui-icon_small oui-icon-pen_concept"
-                    aria-hidden="true"
-                ></span>
-            </button>
         </div>
         <p data-ng-bind="$ctrl.nasha.serviceName"></p>
         <oui-header-tabs>
             <oui-header-tabs-item
                 data-href="{{:: $ctrl.partitionHref() }}"
-                data-active="[$ctrl.partitionHref(), $ctrl.editNameHref(), $ctrl.editDescriptionHref(), $ctrl.editSizeHref()].includes($ctrl.currentHref())"
+                data-active="[$ctrl.partitionHref(), $ctrl.editDescriptionHref(), $ctrl.editSizeHref()].includes($ctrl.currentHref())"
```

Comment:
We should avoid this kind of logic in the HTML (especially polyfills won't be applied)

Reply:
:+1: 
Moved into the routing